docs/consortium/minutes/2018-10-11-minutes/index.html

```diff
@@ -0,0 +1,227 @@
+
+
+  <link href="/css/font.css" rel="stylesheet" type="text/css">
+  <link href="/css/kube.min.css" rel="stylesheet" type="text/css">
+  <link href="/css/kube.legenda.css" rel="stylesheet" type="text/css">
+  <link href="/css/highlight.css" rel="stylesheet" type="text/css">
+  <link href="/css/master.css" rel="stylesheet" type="text/css">
+  <link href="/css/kube.demo.css" rel="stylesheet" type="text/css">
+
+ <link href="/css/custom.css" rel="stylesheet" type="text/css">
+
+  <script src="/js/jquery-2.1.4.min.js" type="text/javascript">
+  </script>
+
+  <script type="text/javascript" src="/js/tocbot.min.js"></script>
+</head>
+
+
+<body class="page-kube">
+  <header> <div class="show-sm">
+    <div id="nav-toggle-box">
+      <div id="nav-toggle-brand">
+        <a href="/">Open Force Field Initiative</a>
+      </div><a data-component="toggleme" data-target="#top" href="#" id="nav-toggle"><i class="kube-menu"></i></a>
+    </div>
+  </div>
+  <div class="hide-sm" id="top">
+    <div id="top-brand">
+      <a href="/" title="home">Open Force Field Initiative</a>
+    </div>
+    <nav id="top-nav-main">
+      <ul>
+       
+       
+    <li><a href="/news/" >News</a></li>
+    
+    <li><a href="https://github.com/openforcefield/" >Code</a></li>
+    
+    <li><a href="/datasets/" >Data</a></li>
+    
+    <li><a href="/science/" >Science</a></li>
+    
+    <li><a href="/publications/" >Publications</a></li>
+    
+    <li><a href="/forcefields/" >Force fields</a></li>
+    
+    <li><a href="/members/" >Members</a></li>
+    
+    <li><a href="/consortium/" >Consortium</a></li>
+    
+    <li><a href="/faq/" >FAQ</a></li>
+    
+      </ul>
+    </nav>
+    <nav id="top-nav-extra">
+      <ul>
+        
+      </ul>
+    </nav>
+  </div>
+
+  
+ </header>
+  <main>
+
+<div id="hero" class="wrap">
+  <h1>October 11, 2018 Governing Board Meetings and notes</h1>
+    <p>October 11, 2018 Governing Board Meetings and notes</p>
+</div>
+<ul id="posts">
+
+ 
+
+<h2 id="agenda">Agenda:</h2>
+
+<ul>
+<li>October meeting draft agenda and start time</li>
+<li>January meeting location</li>
+<li>Maat hiring</li>
+<li>Slack</li>
+<li>Funding processing update/onboarding update</li>
+<li>Involvement with science meetings (subprojects) and project planning</li>
+<li>Discuss blog posts? Yes, just get sub-groups to do brief updates after meetings; better for keeping people up to date.</li>
+</ul>
+
+<h2 id="minutes">Minutes:</h2>
+
+<p>Attending: All PIs except Shirts, and both industry members, Thomas Fox and Katharina Meier</p>
+
+<p>Shared the detailed budget outline Google Doc and went over the personnel who would be involved and their distribution across different locations, $ amounts, etc. Approved were:</p>
```

Comment:
We said we'd post "summarized minutes" so my feeling is that it is not necessary (nor helpful) to go quite to this level of detail. I will suggest a couple specific edits that I think would help. (Mostly people aren't going to find the minutiae that exciting so it actually can be MORE useful to people by having less.)